js/menu.js

```diff
@@ -20,3 +20,169 @@ hamburgerIcon.addEventListener('click', appear);
+const workSections = document.querySelector('.works');
+const modal = document.querySelector('.modal-example');
+const modalImg = document.querySelector('.modal-img');
+const company = document.querySelector('.company');
+const role = document.querySelector('.role');
+const year = document.querySelector('.year');
+
+// create the array of project's objects:
+const projectsData = [
+  {
+    id: 'card1',
+    name: 'Tonic',
+    description: 'A daily selection of privately personalized reads, no accounts or sign-ups required.',
+    imageLink: './img/Snapshoot-Portfolio-1.png',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    role: 'Back End Dev',
+    company: 'CANOPY',
+    year: '2015',
+    sourceLink: 'https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio',
+    demoLink: 'https://ivonnebenitesrodriguez.github.io/microverse_mobile_portfolio/',
+    icon: 'Counter.svg',
+    seeBtn: 'See Project',
+  },
+  {
+    id: 'card2',
+    name: 'Multi-Post<br/>Stories',
+    description: 'A daily selection of privately personalized reads, no accounts or sign-ups required.',
+    imageLink: './img/Snapshoot-Portfolio-2.png',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    role: 'Full Stack Dev',
+    company: 'Facebook',
+    year: '2015',
+    sourceLink: 'https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio',
+    demoLink: 'https://ivonnebenitesrodriguez.github.io/microverse_mobile_portfolio/',
+    icon: 'Counter.svg',
+    seeBtn: 'See Project',
+  },
+  {
+    id: 'card3',
+    name: 'Facebook 360',
+    description: 'A daily selection of privately personalized reads, no accounts or sign-ups required.',
+    imageLink: './img/Snapshoot-Portfolio-3.png',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    role: 'Back End Dev',
+    company: 'FACEBOOK',
+    year: '2015',
+    sourceLink: 'https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio',
+    demoLink: 'https://ivonnebenitesrodriguez.github.io/microverse_mobile_portfolio/',
+    icon: 'Counter.svg',
+    seeBtn: 'See Project',
+  },
+  {
+    id: 'card4',
+    name: 'Uber Navigation',
+    description: 'A daily selection of privately personalized reads, no accounts or sign-ups required.',
+    imageLink: './img/Snapshoot-Portfolio-4.png',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    role: 'Lead Developer',
+    company: 'UBER',
+    year: '2018',
+    sourceLink: 'https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio',
+    demoLink: 'https://ivonnebenitesrodriguez.github.io/microverse_mobile_portfolio/',
+    icon: 'Counter.svg',
+    seeBtn: 'See Project',
+  },
+];
+
+workSections.innerHTML = '';
+for (let i = 0; i < projectsData.length; i += 1) {
+  // let languages = '';
+  // projectsData[i].technologies.forEach((language) => {
+  //   // workSections.innerHTML += `
+  // // <li><a href='' class='skill' title='${language}'>${language}</a></li>
+  // // `;
+  // });
```

Comment:
- [ ] Kindly make sure to remove commented code from your code base so it looks clean and easy to navigate

Reply:
Hello @Ismailco 👋🏾
Thank you for your kind suggestions, I appreciate it,
I corrected all the changes you suggested, 
thanks again,  🌷
Best regards,